### DISCUSSION TITLE
watermarks - what is the current state about this?

### DISCUSSION THREAD

**Discussion by cgidesign-de** [vladmandic/sdnext#2928] [General]
Do images created with sd.next (version Feb. 2014) contain some kind of automatically created watermarks?
If yes, what imformation is included in those?
If yes, can the creation of watermarks be disabled?

I read in an older DEV post that
"[controllnet is heavily watermarked](note: controlnet appears to be heavily watermarked)"
Is this still the case?

**Comment by vladmandic** [comment]
three different topics
- in any area where model might actively add watermark, sdnext will remove that code that creates watermark.
- if model is trained on heavily watermarked images (which is the case with some controlnets), there is no avoiding them. those are not digital watermarks, but simple image bleeds.
- you do have option to set your digital watermark (disabled by default) in setttings.

**Comment by cgidesign-de** [comment]
Thank you for the answer. I meant topic one - to the others I had found answers here in the discussions.
